class.jetpack.php

```diff
@@ -4899,6 +4910,13 @@ public static function staticize_subdomain( $url ) {
 		// Explode hostname on '.'
 		$exploded_host = explode( '.', $host );
 
+		// Retreive the name and TLD
+		$name = $exploded_host[ count( $exploded_host ) - 2 ];
+		$tld = $exploded_host[ count( $exploded_host ) - 1 ];
+
+		// Rebuild domain excluding subdomains
+		$domain = $name . '.' . $tld;
```

Comment:
This logic is already handled within a conditional right below.  Not sure why it was re-written here as well?